lib/std/mem.zig

```diff
@@ -38,6 +18,60 @@ pub const byte_size_in_bits = 8;
 
 pub const Allocator = @import("mem/Allocator.zig");
 
+/// Stored as a power-of-two.
+pub const Alignment = enum(math.Log2Int(usize)) {
+    @"1" = 0,
+    @"2" = 1,
+    @"4" = 2,
+    @"8" = 3,
+    @"16" = 4,
+    @"32" = 5,
+    @"64" = 6,
```

Comment:
What's the point of these vs using `.fromByteUnits(n)`?

Reply:
it would be e.g. `.@"8"` vs `comptime .fromByteUnits(8)` to get the same behavior. Which one would you rather read and write?

Reply:
I personally would rather the latter, both because using `@""` syntax for something that seems like it should be as general as this seems strange, and because if I wanted to change from `.@"64"` to be 128, I would have to switch over to `.fromByteUnits` anyway - using the latter directly is a smaller diff.